### ISSUE TITLE
Multiple-output Truthtables

### ISSUE THREAD

**Issue by DanseMacabre** [cjdrake/pyeda#141]
Support for multiple-output truthtables generated from a farray, or a list of exprs would be useful.

From a cursory glance, this is not supported by PyEDA.

Would you be interested in a PR implementing this feature, or in implementing it yourself?

**Comment by cjdrake** [comment]
Yeah, that sounds like a great feature. Would love to receive a pull request for this.

**Comment by Abd-Bin** [comment]
was this ever done?
